### PR TITLE
Get the signature when login to wax cloud wallet

### PR DESCRIPTION
**Pull request by hieuwax**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -114,14 +118,19 @@ export class WalletPluginCloudWallet extends AbstractWalletPlugin implements Wal
 
         // Retrieve translation helper from the UI, passing the app ID
         const t = context.ui.getTranslate(this.id)
+        
+        // TODO: The nonce should be generate on the BE side. 
+        const nonce = `${context.appName}-${new Date().getTime()}`
```
Comment (includenull) [inline src/index.ts:123]: The nonce should be passed in by the dapp owner on login not generated here.

Comment (includenull) [reply src/index.ts:123]: I think that would be ideal, the dapp owner can then await a request to their backend before calling the login method.

Comment (aaroncox) [reply src/index.ts:123]: I created a branch of the Session Kit which includes a new `arbitrary` object which can be passed to the login method. I tried to keep it flexible for potential other wallet plugins to take advantage of as well. 

The new login call will look like this:

https://github.com/wharfkit/session/commit/6cbb786471e3856a18dc6e382d14931499485063#diff-5f227b143bbe6372c0905b64e6d1e089e45d22608109c4b0f86a6c385580e21bR476-R480

And wallet plugins can access this data through the `context` on the `arbitrary` field like so:

https://github.com/wharfkit/session/commit/6cbb786471e3856a18dc6e382d14931499485063#diff-5f227b143bbe6372c0905b64e6d1e089e45d22608109c4b0f86a6c385580e21bR460

If this works, I can publish a release containing this update. Feedback welcome!

Comment (hieuwax) [reply src/index.ts:123]: Hi @aaroncox, this is a good way to resolve the "nonce" issue.
Thank you so much,

**Review by aaroncox — APPROVED** [review]

File: src/index.ts (hunk 1 above)
Comment (aaroncox) [reply src/index.ts]: Sorry for the delay on this one - this is released as 1.3.0

**Pull request merged**